index.html

```diff
@@ -1214,459 +1400,168 @@ <h3>Revocation</h3>
       </section>
       <section>
         <h3>Fitness for Purpose</h3>
+
         <ul>
-          <li>The custom properties in the <a>credential</a> should be
-          appropriate for the verifier's purpose. For example if an
-          verifier needs to determine that a subject is older than 21
-          years of age, they may accept claims of specific birthdate or
-          abstract properties such as <code>ageOver</code>.</li>
-          <li>The issuer is trusted by the verifier to make the claims
-          at hand. For example, Fast Food Resturant A will not be trusted
-          to make a claim that an individual may enjoy a lifetime 10%
-          discount to its competitor Fast Food Restaurant B.</li>
-          <li>If the <a>issuer</a> has placed any policy information about
-          the use of the <a>credential</a>, e.g. intended
-          <a>verifiers</a>, expiration date, etc., that this
-          policy is adhered to.</li>
-          <li>If the <a>holder</a> has placed any policy information about
-          the use of the <a>credential</a>, e.g. intended
-          <a>verifiers</a>, restricted usage rights, etc., that this
-          policy is adhered to.</li>
+          <li>The custom properties in the <a>credential</a> MUST be
+          appropriate for the <a>verifier</a>'s purpose. For example if a
+          verifier needs to determine that a <a>subject</a> is older than 21
+          years of age, they may accept claims of specific
+          <code>birthdate</code> or abstract properties such as
+          <code>ageOver</code>.</li>
+          <li>The <a>issuer</a> is trusted by the <a>verifier</a> to make the
+          claims at hand. For example, a franchised Fast Food resturant
+          location will trust discount coupon claims made by the
+          corporate headquarters of the franchise.</li>
+          <li>Any policy information expressed by the <a>issuer</a>
+          in the <a>verifiable credential</a> SHOULD be enforced. For example,
+          the issuer may limit the use of the claims to
+          specific <a>verifiers</a>, certain <a>holder</a> age ranges,
+          or during certain dates.</li>
+          <li>Any policy information expressed by the <a>holder</a>
+          in the <a>verifiable presentation</a> SHOULD be enforced. For example,
+          the holder may limit the use of the claims to
+          specific <a>verifiers</a> or for specific purposes
+          (such as authentication, but not data mining).</li>
```

Comment:
I can't remember if we decided to reword these "SHOULD" and "MUST" statements because they weren't testable or not. Just making a note here in case this was unintentional and needs to be addressed.

Reply:
They should be MUSTs, since they must be enforced by all standard's conforming verifiers. The issuer or holder place policy information in the VC or VP precisely to tell the verifier what it must do.
Note that this does not stop a verifier's implementation ignoring all policy fields and doing what it wants. But in this case it is non-conformant to the standard, and therefore can do what it likes. But the standard does not condone this nor support it.

Reply:
Fixed in 2110618f0e7454bc6357f46942872dc316425c3a.